src/components/pages/novu-framework/code/create-custom-element/create-custom-element.jsx

```diff
@@ -160,10 +160,9 @@ export default function createCustomElement({
 
     if (regex.test(value)) {
       const matchedValue = value.match(regex);
-
       return (
         <>
-          {value[0] === ' ' && ' '}
+          {matchedValue.index > 0 && ' '.repeat(matchedValue.index)}
```

Comment:
Fix to ensure the `skip` is aligned correctly. This pads the string to the left appropriately.